actions/java-gradle-build-jib/action.yaml

```diff
@@ -86,21 +82,34 @@ runs:
       with:
         images: ${{ inputs.image-name }}
         tags: |
-          event=push,type=raw,value=${{ inputs.image-tags }}
-          event=pr,type=raw,value=${{ inputs.image-tags }}
-          event=tag,type=semver,pattern={{ version }}
+          event=tag,type=semver,pattern={{version}}
+          event=push,type=raw,value=pipeline-${{ github.run_id }}-git-{{sha}},enable=${{ github.ref_type != 'tag' }}
+        flavor: |
+          latest=false
 
     - name: Build Docker image
       run: |
-        ./gradlew ${{ inputs.subproject && format('{0}:', inputs.subproject) || '' }}jib \
-          --info --stacktrace \
-          --image=${{ inputs.image-name }} \
-          -Djib.from.platforms=${{ inputs.platforms }} \
-          ${{ inputs.jib-from-image && format('-Djib.from.image={0}', inputs.jib-from-image) || '' }} \
-          ${{ inputs.class && format('-Djib.container.mainClass={0}', inputs.class) || '' }} \
-          -Djib.to.image=${{ inputs.full-image-name }} \
-          -Djib.to.tags=$(echo "${{ steps.meta.outputs.tags }}" | tr ',' '\n' | sed 's/.*://') \
-          ${{ inputs.gradle-refresh-dependencies == 'true' && '--refresh-dependencies' || '' }} \
-          ${{ inputs.allow-insecure-registries == 'true' && '-Djib.allowInsecureRegistries=true' || '' }}
+        if [ "${{ github.ref_type }}" != "tag" ]; then
+          ./gradlew ${{ inputs.subproject && format('{0}:', inputs.subproject) || '' }}jib \
+            --info --stacktrace \
+            --image=${{ inputs.image-name }} \
+            -Djib.from.platforms=${{ inputs.platforms }} \
+            ${{ inputs.jib-from-image && format('-Djib.from.image={0}', inputs.jib-from-image) || '' }} \
+            ${{ inputs.class && format('-Djib.container.mainClass={0}', inputs.class) || '' }} \
+            -Djib.to.image=${{ inputs.full-image-name }}$(echo ${{ steps.meta.outputs.tags }} | grep 'pipeline' | sed 's/.*\(:.*\)/\1/') \
+            ${{ inputs.gradle-refresh-dependencies == 'true' && '--refresh-dependencies' || '' }} \
+            ${{ inputs.allow-insecure-registries == 'true' && '-Djib.allowInsecureRegistries=true' || '' }}
+        else
+          ./gradlew ${{ inputs.subproject && format('{0}:', inputs.subproject) || '' }}jib \
+            --info --stacktrace \
+            --image=${{ inputs.image-name }} \
+            -Djib.from.platforms=${{ inputs.platforms }} \
+            ${{ inputs.jib-from-image && format('-Djib.from.image={0}', inputs.jib-from-image) || '' }} \
+            ${{ inputs.class && format('-Djib.container.mainClass={0}', inputs.class) || '' }} \
+            -Djib.to.image=${{ inputs.full-image-name }} \
```

Comment:
can you eliminate the if and rather do sth like `-[Djib.to](http://djib.to/).image=${{ image-name}}:{{ tag }} ${{ is-tag == 'true' && '-[Djib.to](http://djib.to/).tags=latest' || '' }}`? That way the other parameters stay the same

Reply:
Is this ok for you 
```
-Djib.to.image=${{ inputs.full-image-name }}$(echo ${{ steps.meta.outputs.tags }} | sed 's/.*\(:.*\)/\1/') \
$([[ "${{ github.ref_type }}" == "tag" ]] && echo "-Djib.to.tags=latest") \
```

Reply:
I just tested it and it works